docs/configuration/aws.md

```diff
@@ -69,3 +71,41 @@ serviceAccount:
   annotations:
     eks.amazonaws.com/role-arn: arn:aws:iam::123456789012:role/S3Access
 ```
+
+## AWS Cloud Cost Configuration
+
+To configure OpenCost for your AWS account, create an Access Key for the OpenCost user who has access to the Cost and Usage Report (CUR). Navigate to the [IAM Management Console dashboard](https://console.aws.amazon.com/iam), and select *Access Management > Users*. Find the OpenCost user and select *Security Credentials > Create Access Key*. Note the Access Key ID and Secret access key.
```

Comment:
How would I add multiple AWS accounts as a user? cc @AjayTripathy @Sean-Holcomb

Reply:
You can add multiple cloud accounts in the `cloud-integration.json` since credentials are in arrays. This showed up in testing, but it may be an awkward experience given most OpenCost deployments are single-cluster. 

https://opencost-website-git-fork-mattray-cloudcosts-kubecost.vercel.app/docs/configuration/#cloud-costs

Reply:
I may just add that link anyway, since you can deploy a multicluster / multicloud opencost?